packages/devtools_app/lib/src/utils.dart

```diff
@@ -130,7 +130,7 @@ void executeWithDelay(Duration delay, void callback(),
   }
 }
 
-Future<void> delayForBatchProcessing({int micros = 0}) async {
+Future<void> delayForBatchProcessing({int micros = 100}) async {
   // Even with a delay of 0 microseconds, awaiting this delay is enough to free
```

Comment:
this comment is now obsolete. did something change that made micros=0 insufficient? 100 microseconds is still tiny.